OpenSim/Common/Scale.h

```diff
@@ -96,15 +96,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Scale, Object);
 
     void getScaleFactors(SimTK::Vec3& aScaleFactors) const;
     SimTK::Vec3& getScaleFactors() { return _scaleFactors; }
+#ifndef SWIG    
     void getScaleFactors(double rScaleFactors[]){   // A variant that uses basic types for use by GUI
         getScaleFactors(SimTK::Vec3::updAs(rScaleFactors));
     }
-
+#endif
     void setScaleFactors(const SimTK::Vec3& aScaleFactors);
+#ifndef SWIG
     void setScaleFactors(const double aScaleFactors[]){ // A variant that uses basic types for use by GUI
         setScaleFactors(SimTK::Vec3::getAs(aScaleFactors));
     }
-
+#endif
```

Comment:
Same comment. This method was likely introduced for GUI use, I'd remove it rather than idndef SWIG it.